jimpass/managers/bitwarden.py

```diff
@@ -66,15 +66,26 @@ def get_session(self) -> str:
         """ Get the key holding the session hash """
         code, stdout = srun("keyctl request user bw_session")
         if code != 0 or not stdout:
-            code, passwd = rofi(
-                prompt='Bitwarden Master Password',
-                options=[
-                    'password'
-                ],
-                args={
-                   'lines': 0
-                }
-            )
+            if self.config['runner'] == "wofi":
```

Comment:
see my comment on `util.py`. The `runner` argument can be obtained rather like this:
```python
self.config.get('runner', 'rofi')
```
That way we have a default `rofi` value for the hypothetical `runner` argument, prevent repeated code, and keeps things DRY

Reply:
it seems there are like 3 places to set a default value - int the function arguments, in cli.py, and in this comment (where I don't see anywhere else in the codebase a `.get()`). Is it enough to set it in the default config and the function argument?

Reply:
Yeah.. since you put it in the default config and parameter it's enough.